### ISSUE TITLE
Mise à jour de rules.csv / Update of rules.csv

### ISSUE THREAD

**Issue by vinber** [PanierAvide/Covid_enseignes#197]
Vérification à faire sur la base du changement du 28 novembre (https://www.gouvernement.fr/info-coronavirus/confinement)

- [x] alcohol
- [x] bakery
- [x] bank
- [x] beauty
- [x] cafe
- [x] car_rental
- [x] car_repair
- [x] chocolate
- [x] clothes
- [x] computer
- [x] doityourself
- [x] electronics
- [x] electronics_repair
- [x] employment_agency
- [x] farm
- [ ] fast_food
- [ ] flower
- [x] frozen_food
- [x] fuel
- [x] funeral_directors
- [x] furniture
- [x] garden_centre
- [x] hairdresser
- [x] hearing_aids
- [ ] hotel
- [ ] insurance
- [x] laundry
- [x] library
- [x] mobile_phone
- [x] optician
- [x] pet
- [x] pharmacy
- [ ] post_office
- [ ] restaurant
- [x] shoes
- [x] sports
- [x] stationery
- [ ] supermarket
- [x] variety_store

**Comment by nlehuby** [comment]
Il y a encore pas mal d'enseignes qui n'ont pas mis à jour leur site pour indiquer une éventuelle réouverture, donc il faudra refaire une passe (peut-être juste sur les `closed` / `partial`) même si c'est coché.

**Comment by vinber** [comment]
yep, je t'ai envoyé un csv car je ne suis pas sur de maitriser les méandres de github :)

**Comment by vinber** [comment]
euh, en fait je suis sur de ne **pas** maîtriser les méandres de github.
